github/komand_github/actions/add_issue_label/action.py

```diff
@@ -0,0 +1,25 @@
+import komand
+from .schema import AddIssueLabelInput, AddIssueLabelOutput, Input, Output, Component
+# Custom imports below
+
+
+class AddIssueLabel(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='add_issue_label',
+                description=Component.DESCRIPTION,
+                input=AddIssueLabelInput(),
+                output=AddIssueLabelOutput())
+
+    def run(self, params={}):
+        if params.get('organization') and params.get('repository'):
+            g = self.connection.github_user
+            issue = g.get_organization(params.get('organization')).get_repo(params.get('repository')).get_issue(int(params.get('issue_number')))
+        else:
+            g = self.connection.user
+            issue = g.get_repo(params.get('repository')).get_issue(int(params.get('issue_number')))
+
+        issue_params = params.get("label")
+        issue = issue.add_to_labels(issue_params)
+        return {}
```

Comment:
We really should return an object, a status, or similar so workflow decisions can be mode off the output.

Same for other actions that are missing output. Needs a plugin spec output schema, updated documentation table for output and example JSON output in `help.md`.

Reply:
@jschipp-r7 The function `[add_to_labels](https://pygithub.readthedocs.io/en/latest/github_objects/Issue.html#github.Issue.Issue.add_to_labels)` returns `None`.

Should I add a try/expect?

Reply:
Oh, sorry, I didn't realize that. You're on the right path then. What we could do is have an output item called `success` or `status` and it returns a boolean. False if there's an exception when the method is called `{ 'return': False }`, True if all good.